src/components/reactcomponents/Table.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+import AddNewIcon from './AddNewIcon';
+
+class Table extends React.Component {
+
+  constructor(props) {
+    super(props);
+      this.state = {
+        jsonData: this.props.jsonData,
+        sortOrder: {
+          id: true,
+          author: true,
+          title: true,
+          year: true,
+          citationCount: true
+        }
+      }
+    }
+
+  render() {
+    return (
+      <div>
+        <table>
+          <thead>
+            <tr>
+              <th onClick={ () => this.sortJsonDataByKey('doi') }> DOI </th>
+              <th onClick={ () => this.sortJsonDataByKey('author') }> Author </th>
+              <th onClick={ () => this.sortJsonDataByKey('title') }> Title </th>
+              <th onClick={ () => this.sortJsonDataByKey('year') }> Year </th>
+              <th onClick={ () => this.sortJsonDataByKey('citationCount') }> Citations </th>
```

Comment:
Do the spaces from `> Citations </th>` actually show up?  If not you can remove the spaces wrapping `Citations` and do the same for the other lines